game/notes.cc

```diff
@@ -43,35 +43,35 @@ double Note::scoreMultiplier() const {
 }
 
 double thresholdForFullScore() {
-	switch(GameDifficulty(config["game/difficulty"].i())){
+	switch(GameDifficulty(config["game/difficulty"].ui())){
 		case GameDifficulty::PERFECT:
```

Comment:
Is the fall through here desired? Maybe add //fallthrough or //falls through

Reply:
can't see any fall through here, what are you referring to?